src/humanize/__init__.py

```diff
@@ -37,6 +38,7 @@
     "naturaldelta",
     "naturalsize",
     "naturaltime",
+    "precisetime",
     "ordinal",
     "precisedelta",
```

Comment:
Put in alphabetical order:
```suggestion
    "ordinal",
    "precisedelta",
    "precisetime",
```